CMakeLists.txt

```diff
@@ -236,6 +237,12 @@ if (ENABLE_HIP)
   set(raja_depends
     ${raja_depends}
     hip)
+  if(ENABLE_ROCTX)
+    include(FindRoctracer)  
```

Comment:
I think you should move the FindRoctracer call into `cmake/SetupPackages.cmake`

Reply:
Done!